classes/class_opensimplexnoise.rst

```diff
@@ -34,6 +35,23 @@ This resource allows you to configure and sample a fractal noise space. Here is
     print(noise.get_noise_3d(0.5, 3.0, 15.0))
     print(noise.get_noise_4d(0.5, 1.9, 4.7, 0.0))
 
+ .. code-tab:: csharp
+
+    OpenSimplexNoise noise = new OpenSimplexNoise();
+
+    // Configure
+    System.Random random = new System.Random(); 
+    noise.Seed = random.Next(); 
```

Comment:
So that the behavior is the same, and for simplicity:

```suggestion
    noise.Seed = GD.Randi(); 
```

Reply:
Ah, thanks! I tried using `GD.RandI()` and when that didn't work then assumed it didn't exist in C#.